Altis_Life.Altis/stringtable.xml

```diff
@@ -4280,7 +4280,7 @@
         <Key ID="STR_NOTF_notEnoughtToGive">
             <Original>You don't have that much to give!</Original>
             <Czech>Nemusíte to hodně dát!</Czech>
-            <French>Vous n'avez pas autant sur vous !</French>
+            <French>Vous n'en avez pas assez à donner !</French>
```

Comment:
I'm ok with the "n'en", but you could have let "autant sur vous" since it's the same meaning and it's more explicit.